src/functionalTest/kotlin/com/vaadin/gradle/VaadinSmokeTest.kt

```diff
@@ -114,4 +114,18 @@ class VaadinSmokeTest : AbstractGradleTest() {
         build.find("webcomponentsjs/webcomponents-*.js", 2..2)
         build.find("webcomponentsjs/bundles/webcomponents-*.js", 4..4)
     }
+
+    /**
+     * Tests https://github.com/vaadin/vaadin-gradle-plugin/issues/73
+     */
+    @Test
+    fun vaadinCleanDoesntDeletePnpmFiles() {
+        val pnpmLockYaml = testProjectDir.touch("pnpm-lock.yaml")
+        val pnpmFileJs = testProjectDir.touch("pnpmfile.js")
+        val webpackConfigJs = testProjectDir.touch("webpack.config.js")
```

Comment:
I don't think you can easily delete this file - it has the following as comment:

```java
/**
 * This file has been autogenerated as it didn't exist or was made for an older incompatible version.
 * This file can be used for manual configuration will not be modified if the flowDefaults constant exists.
 */
```

Reply:
Hmm, you're right. The file by default does nothing, so there is no harm keeping it around. Also, newer Vaadin should not suddenly start placing logic in there, since there's `webpack.generated.js` for that. I'll fix the PR.